### ISSUE TITLE
Remove WebTorrent support in client in V6

### ISSUE THREAD

**Issue by Chocobozzz** [Chocobozzz/PeerTube#5465]
### Describe the problem to be solved

WebTorrent is a wonderful library that allowed PeerTube to be created five years ago. But due to [many limitations](https://docs.joinpeertube.org/admin-configuration?id=webtorrent-transcoding-or-hls-transcoding) of the library we decided to implement and default to another P2P protocol used in coordination with HLS. This new HLS player works very well, and does not have all the limitations of the WebTorrent player. It also allowed us to implement live streaming in PeerTube.

We decided to maintain the WebTorrent player for several years, but it leads to a bigger bundle size for a bad user experience (slow start time, hacky resolution change etc.). Moreover the P2P aspect of WebTorrent is not really used since most of new and popular videos are using HLS now.

It's the reason why we want to remove WebTorrent support in PeerTube V6 (end of 2023). It will allow us to clean and improve the player code.

Removing WebTorrent support doesn't mean that old videos transcoded for WebTorrent won't work anymore. We'll be able to play them using HTML directly, without P2P. Auto resolution change won't be supported either. PeerTube will still generate torrents for video files so external tool can still use the BitTorrent protocol to stream files. But we'll remove redundancy support of WebTorrent videos.

Transcoding options will stay the same: "WebTorrent transcoding" will just be renamed (not sure about the new term, something like "Raw video file transcoding" or "HTML transcoding"). "WebTorrent transcoding" is just a MP4 video file with a MOOV atom at the beginning so it's easy to play it with any video player (including HTML `<video>` element). We may still need such file for downloads, HTML `<video>` element and to play audio only (since the HLS player doesn't support it: https://github.com/Chocobozzz/PeerTube/issues/3281).

TLDR:
 * Thanks WebTorrent to exist :heart: 
 * WebTorrent support will be removed from PeerTube client in v6 (end of 2023)
 * Torrent files will still be generated by PeerTube
 * Old WebTorrent videos will still be played by PeerTube using a classic HTML player without P2P
 * WebTorrent transcoding will still exist with the same transcoding options to support download, audio only player, HTML `<video>` element: only the label will change
 * Redundancy support of WebTorrent videos will be removed
